terraform/concourse/elb.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_elb" "concourse" {
   name            = "${var.env}-concourse"
-  subnets         = ["${var.subnet0_id}"]
+  subnets         = ["${split(",", var.infra_subnet_ids)}"]
```

Comment:
Concourse is only deployed in the first infra subnet. Does it make sense to configure all subnets in the ELB?